pydantic_ai_slim/pydantic_ai/models/test.py

```diff
@@ -45,6 +45,55 @@ class _WrappedToolOutput:
     value: Any | None
 
 
+@dataclass
+class TestToolCallPart:
```

Comment:
Did you see `DeltaToolCall` in `models/function.py`, and the uses in `test_streaming.py`? I'd prefer to use that same approach here (and add to it if necessary)

Reply:
Looks like that is doing something very different, this is just triggering the use of existing TestModel behaviour to create `ToolCallPart`s, where as `DeltaToolCall` represents a tool call part?